### PR TITLE
Updated documentation about replicating SIGIR Forum results

### PR DESCRIPTION
**Pull request by lintool**
Reorganized, added link from main page.

### CODE REVIEW

**Comment by lintool** [comment]
Closing. Repushed branch to add to PR already open in #535

**Comment by lintool** [comment]
oops. wrong issue.

File: docs/experiments-forum2018.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+python src/main/python/fine_tuning/run_batch.py --collection robust04 --model bm25+rm3 --n 44 --run --use_drr_fold
+```
+
+BM25+AxiomaticReranking Robust04 (runs + eval + print results):
+
+```
+python src/main/python/fine_tuning/run_batch.py --collection robust04 --model bm25+axiom --n 44 --run --use_drr_fold
+```
+
+Change `robust04` to `cw0b9` for ClueWeb09b results.
```
Comment (Peilin-Yang) [inline docs/experiments-forum2018.md]: Since we did not run `cw09b` (not `cw0b9`) I think we can remove this part

**Comment by lintool** [comment]
@Peilin-Yang Best params for 2fold and 5fold already checked in as part of https://github.com/castorini/Anserini/pull/546 - already referenced in the docs.

**Pull request merged**